float64[]           building_x
float64[]           building_y
float64             flight_altitude
float64             building_distance
float64             trajectory_resolution
string		    wp_file_path
string		    coord_sys_text		
---
float64[]           traj_wp_x
float64[]           traj_wp_y
float64[]           traj_wp_z
float64[]           traj_wp_yaw
